RealmSwift/Sync.swift

```diff
@@ -500,7 +518,7 @@ extension SyncUser {
      - warning: NEVER disable SSL validation for a system running in production.
      */
     public func configuration(realmURL: URL? = nil, fullSynchronization: Bool = false,
-                              enableSSLValidation: Bool = true, urlPrefix: String? = nil) -> Realm.Configuration {
+                              enableSSLValidation: Bool, urlPrefix: String? = nil) -> Realm.Configuration {
```

Comment:
[minor] Isn't this a breaking change?

Reply:
It changes which of the two overloads are called if neither `enableSSLValidation` nor `serverValidationPolicy` are supplied, but they both do the same thing in that case, and doesn't change anything if either of them are supplied. Anything which previously compiled should still compile and do the same thing.